teamB/shinInHuck/myapp/.env.sample

```
@@ -0,0 +1,11 @@
+DATABASE_URL="mysql://username:password@127.0.0.1:3306/database_name"
+
+DATEBASE_URL = URL
+
+TYPEORM_CONNECTION = mysql
+TYPEORM_HOST = 127.0.0.1
+TYPEORM_USERNAME = root
+TYPEORM_PASSWORD = 'Tlm753!@#dsl'
+TYPEORM_DATABASE = westagram
+TYPEORM_PORT = 3306
+TYPEORM_LOGGING =TRUE 
```

- 배포 및 상용화의 목적으로 Github main 브랜치에 올리는 코드에 서버의 다양한 기록들을 확인해볼 수 있는 기능인 logging 이 True가 되어 있어서는 안됩니다. 해당 내용은 삭제하거나 false로 수정해주세요 !
- 더불어 .env.sample의 경우 타인이 해당 서버를 사용할 때 설명과 같은 역할을 하기 때문에 어떤 것을 넣을지 이해할 수 있도록 YOUR NAME등 과 같은 설명이 들어가야합니다.

```suggestion
TYPEORM_CONNECTION = mysql
TYPEORM_HOST = 127.0.0.1
TYPEORM_USERNAME = YOUR NAME
TYPEORM_PASSWORD = YOUR PW
TYPEORM_DATABASE = YOUR DB
TYPEORM_PORT = 3306
TYPEORM_LOGGING =TRUE 
```